files/galaxy/tpv/tools.yml

```diff
@@ -1437,3 +1437,6 @@ tools:
     scheduling:
       require:
         - singularity
+
+  toolshed.g2.bx.psu.edu/repos/galaxyp/fragpipe/fragpipe/.*:
+    cores: 8
```

Comment:
By default you only get 4GB

Reply:
Okay, I should increase the memory in the pull request too, then. Do you have an estimate for how long jobs requesting 8-cores, 256GB memory would take to queue?

Reply:
I don't know. You can try. But I would start with 56 or so!

Reply:
Okay, let's try that. Can always bump it up later.